python/example_code/ssm/ssm_getting_started.py

```diff
@@ -0,0 +1,235 @@
+                command_id = self.document_wrapper.send_command(
+                    instance_ids=[instance_id]
+                )
+
+                self.document_wrapper.wait_command_executed(
+                    command_id=command_id, instance_id=instance_id
+                )
+
+                print("-" * 88)
+                print(
+                    "Lets get the time when the specific command was sent to the specific managed node"
+                )
+                q.ask("Please hit Enter")
+
+                self.document_wrapper.list_commands(instance_id=instance_id)
+
+            print("-" * 88)
+            print("-" * 88)
+            print(
+                """
+Now we will create a  Systems Manager OpsItem.
+An OpsItem is a feature provided by the Systems Manager service.
+It is a type of operational data item that allows you to manage and track various operational issues,
+events, or tasks within your AWS environment.
+
+You can create OpsItems to track and manage operational issues as they arise.
+For example, you could create an OpsItem whenever your application detects a critical error
+or an anomaly in your infrastructure.
+            """
+            )
+            q.ask("Please hit Enter")
+
+            self.ops_item_wrapper.create(
+                title="Disk Space Alert",
+                description="Created by the Systems Manager Python (Boto3) API",
+                source="EC2",
+                category="Performance",
+                severity="2",
+            )
+
+            print("-" * 88)
+            print("-" * 88)
+            print(f"Now we will update  the OpsItem {self.ops_item_wrapper.id}")
+            q.ask("Please hit Enter")
+
+            self.ops_item_wrapper.update(
+                title="Disk Space Alert",
+                description=f"An update to {self.ops_item_wrapper.id}",
+            )
+
+            print(
+                f"Now we will get the status of the OpsItem {self.ops_item_wrapper.id}"
+            )
+            q.ask("Please hit Enter")
+
+            self.ops_item_wrapper.describe()
+
+            print(f"Now we will resolve the OpsItem {self.ops_item_wrapper.id}")
+            q.ask("Please hit Enter")
+
+            self.ops_item_wrapper.update(status="Resolved")
+
+            print("-" * 88)
+            print("-" * 88)
+            if q.ask(
+                "Would you like to delete the Systems Manager resources? (y/n)",
+                q.is_yesno,
+            ):
+                print("You selected to delete the resources.")
+                self.cleanup()
+            else:
+                print("The Systems Manager resources will not be deleted")
+
+            print("-" * 88)
+            print("This concludes the Systems Manager SDK Getting Started scenario.")
+            print("-" * 88)
+
+        except Exception:
+            self.cleanup()
+            raise
+
+    def cleanup(self):
+        self.maintenance_window_wrapper.delete()
```

Comment:
Do we need any kind of logging here that these resources were deleted?

Reply:
I added logging.